lib/core-integration/src/Test/Integration/Scenario/API/Shelley/Migrations.hs

```diff
@@ -196,18 +197,21 @@ spec = describe "SHELLEY_MIGRATIONS" $ do
                         (errMsg404NoWallet $ sourceWallet ^. walletId)
                     ]
 
+
     it "SHELLEY_CREATE_MIGRATION_PLAN_04 - \
         \Cannot create a plan for a wallet that only contains dust."
```

Comment:
I guess this technically might not be a dust wallet?, but it should at least be a wallet of `Freerider`s

Reply:
For a wallet of just ada values, we **_could_** define "dust" as an ada value that can't be an input to a singleton transaction.

Each value would need to be less than **_output minimum ada quantity_** + **_base cost of tx_** + **_marginal cost of tx input_**.